storm-core/test/clj/backtype/storm/nimbus_test.clj

```diff
@@ -846,16 +846,6 @@
                   NIMBUS-SLOTS-PER-TOPOLOGY 8}]
     (letlocals
       (bind topology (thrift/mk-topology
-                        {"1" (thrift/mk-spout-spec (TestPlannerSpout. true) :parallelism-hint 0 :conf {TOPOLOGY-TASKS 1})}
```

Comment:
Removed this test because it now throws different exception and this is tested in the new unit test I added for `TopologyBuilder`. Actually, I tried to change the expected exception, but this did not fix this test for me (the test fails if I change `InvalidTopologyException` to `IllegalArgumentException`). Don't know why. But I am still new to Clojure and Clojure tests.